dendrite/pipeline.yml

```diff
@@ -38,14 +38,14 @@ steps:
 
   - command:
       - docker build -t complement-dendrite -f build/scripts/Complement.Dockerfile .
-      - "wget -N https://github.com/matrix-org/complement/archive/master.tar.gz && tar -xzf master.tar.gz"
-      - "cd complement-master && COMPLEMENT_BASE_IMAGE=complement-dendrite:latest go test -v -tags=\"dendrite_blacklist\" ./tests"
+      - "wget -O - https://github.com/matrix-org/complement/archive/HEAD.tar.gz | tar -xz --strip-components=1 -C complement"
```

Comment:
the `--strip-components=1 -C complement` means that we don't care about whether the folder in the tarball is named `complement-master` or named after whatever `HEAD` happens to point to right now.

Reply:
FYI you'll need to `mkdir complement` as `tar` expects the directory to exist before it writes into it.